src/Makefile

```diff
@@ -30,7 +30,7 @@ misc.o \
 structs.o \
 version.o
 
-EXECUTABLES = fiberassign
+EXECUTABLES = fiberassign_exec
```

Comment:
For consideration:  There are enough science collaboration fiberassign users out there that this could create confusion by renaming `fiberassign` to `fiberassign_exec` while having the python wrapper be called `run-fiberassign`.  If we are going to rename anything, I'd be inclined to make the python script be `fiberassign` and rename the C++ executable.  i.e. leave `fiberassign` as the name of the thing that the end-user would run.

Reply:
I keep `fiberassign` for the python wrapper and leave `fiberassign_exec` for the C++ executable.